.github/workflows/a11y.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: "16"
+          node-version-file: '.nvmrc'
```

Comment:
:+1: Excellent.

If this is a problem for the NPM version that is bundled with specific node versions should we also ensure that we use same version in [`docker-compose.yml`](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/main/docker-compose.yml#L5)? It currently specifies `node:16`.

This may also be a sign that we should at least add comments in `.nvmrc` and `docker-compose.yml` explaining that developers should keep versions in sync between the two files.